.kokoro/runintegrationtests.sh

```diff
@@ -26,31 +13,32 @@ export GOOGLE_APPLICATION_CREDENTIALS="$KOKORO_KEYSTORE_DIR/73609_cloud-sharp-je
 export REQUESTER_PAYS_CREDENTIALS="$KOKORO_KEYSTORE_DIR/73609_gcloud-devel-service-account"
 
 # Non-coverage run doesn't need any extra flags
-SCRIPT_FLAGS=
+script_flags=
+report_flags=
 
 # If we have any previous coverage runs, remove them, regardless
 # of whether we're about to create any.
 rm -rf coverage
 
 if [[ -f "$KOKORO_KEYSTORE_DIR/73609_codecov-token" ]]
 then
-  CODECOV_TOKEN=$(cat "$KOKORO_KEYSTORE_DIR/73609_codecov-token")
-  SCRIPT_FLAGS=--coverage
-  choco install codecov
+  export CODECOV_TOKEN=$(cat "$KOKORO_KEYSTORE_DIR/73609_codecov-token")
+  script_flags=--coverage
+  report_flags=--commit $KOKORO_GITHUB_COMMIT --build $KOKORO_BUILD_NUMBER
 fi
 
 # Build the libraries and run unit tests, optionally with coverage.
-./build.sh $SCRIPT_FLAGS
+./build.sh $script_flags
 
-maybe_upload_coverage unittests
+./createcoveragereport.sh --upload unittests $report_flags
```

Comment:
We only want to do the upload when codecov-token exists.

In order to simplify this, we probably want *4* flags to createcoveragereport.sh:

--upload: not followed by anything. (It's just a boolean toggle) This is the "master switch"; if this isn't flipped, we don't do anything
--upload_commit - the commit
--upload_build - the build number
--upload_reportname - the report name (--flags)

So:
- Set report_flags to --upload_commit $KOKORO_GITHUB_COMMIT --upload_build $KOKORO_BUILD_NUMBER unconditionally
- In the if block a few lines above, set report_flags to "$report_flags --upload"
- The createcoveragereport.sh calls pass in `$report_flags --upload_reportname unittests` (or integrationtests)

Reply:
Ah yes, in case we want at some point to generate the report without uploading, maybe locally for instance. Yep, get it.

Reply:
And on Unix, where dotCover doesn't work :)

Reply:
True, neither does codecov.exe, totally forgot about that. Pushing now.